### PR TITLE
Well.headI/J can change on timesteps

### PR DESCRIPTION
**Pull request by jokva**
Well.headI and Well.headJ was assumed to be fixed on first declaration,
and that all subsequent changes of this position was an error. This
turns out not to be the case, so the I/J positions are made dynamic.

Solves https://github.com/OPM/opm-parser/issues/984

### CODE REVIEW

File: opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp [hunk 1]
```diff
@@ -358,8 +358,22 @@ namespace Opm {
...
+
+            if( currentWell->getHeadI() != headI ) {
+                std::string msg = "HEAD_I changed for well " + currentWell->name();
+                this->m_messages.warning(keyword.getFileName(), msg, keyword.getLineNumber());
+                currentWell->setHeadI( currentStep, headI );
+            }
+
+            if( currentWell->getHeadJ() != headJ ) {
+                std::string msg = "HEAD_J changed for well " + currentWell->name();
+                this->m_messages.warning( keyword.getFileName(), msg, keyword.getLineNumber() );
```
Comment (pgdr) [inline opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp]: `warning → info`?

Comment (jokva) [inline opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp]: Done.

**Comment by jokva** [comment]
I'm also addressing https://github.com/OPM/opm-parser/issues/986 in this PR.

**Comment by joakim-hove** [comment]
jenkins build this with downstreams opm-output=143 opm-core=1109 please

**Comment by OPMUSER** [comment]
Joka,

What does this error message mean?

ERROR the keyword COMPLUMP is not supported used when C1 or C2 in 
WELOPEN have values


The WELOPEN statement is:

WELOPEN
'PAS-GP01'  OPEN                          /
'PAS-GP01'  OPEN     0   0    0    1    1 /
'PAS-GP01'  OPEN     0   0    0    2    2 /
/

ThunderBird Signature File

Regards, David Baxendale Email: David.Baxendale@gmail.com

------------------------------------------------------------------------

**Comment by jokva** [comment]
jenkins build this with downstreams opm-output=143 opm-core=1109 please

**Comment by jokva** [comment]
Hi David,

From a quick scan it looks like it's the COMPLUMP keyword that isn't well supported, and this error is raised because we can't really guarantee the correct configuration from WELOPEN. There might be something else, but this is what it looks like right now. I'll have a closer look tomorrow.

**Comment by OPMUSER** [comment]
Jokva,

Did you have a chance to look at this in more detail? If I know what is 
the cause I can perhaps modify the input deck to get the run working.
ThunderBird Signature File

Regards, David Baxendale Email: David.Baxendale@gmail.com

------------------------------------------------------------------------

**Comment by jokva** [comment]
It seems to me like we don't really do anything with the COMPLUMP keyword, so using values in WELSPECS' completion fields don't really make sense.


I've been busy with another task that's shaping up nicely, so I expect to look at this early next week, and hopefully have it in order by next weekend (unless I'm seriously underestimating this task).

________________________________

**Pull request merged**